docs/_layouts/default.html

```diff
@@ -25,6 +25,19 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <script>
```

Comment:
We could add a simple comment *before* the script to tell what the following snippet is for.

Reply:
done :) a little comment hurt nobody